pkgs/applications/audio/audacity/default.nix

```diff
@@ -167,18 +166,20 @@ in stdenv.mkDerivation rec {
     "-DDISABLE_DYNAMIC_LOADING_FFMPEG=ON"
     "-Daudacity_conan_enabled=Off"
     "-Daudacity_use_ffmpeg=loaded"
+    "-Daudacity_has_vst3=Off"
 
     # RPATH of binary /nix/store/.../bin/... contains a forbidden reference to /build/
     "-DCMAKE_SKIP_BUILD_RPATH=ON"
   ];
 
+  LD_LIBRARY_PATH = "/build/source/build/utils/";
```

Comment:
What is this for?

Reply:
To work around the following error:

```
[ 57%] Generating LightThemeAsCeeCode.h...
../../utils/image-compiler: error while loading shared libraries: lib-theme.so: cannot open shared object file: No such file or directory
```

Reply:
The `/build` and `/source` parts of the path should not be hardcoded. Do
```
preBuild = ''
    export LD_LIBRARY_PATH=$PWD/utils
'';
```
Or check if one can remove the `"-DCMAKE_SKIP_BUILD_RPATH=ON"` option that is set above.

Reply:
Done.